tests/functional/c/confusing_consecutive_elif.py

```diff
@@ -0,0 +1,11 @@
+# pylint: disable=missing-module-docstring, missing-function-docstring
+
+def check_config(machine, old_conf, new_conf):
+    if old_conf:
+        if not new_conf:
+            machine.disable()
+        elif old_conf.value != new_conf.value:
+            machine.disable()
+            machine.enable(new_conf.value)
+    elif new_conf:  # [confusing-consecutive-elif]
```

Comment:
Could you also add an example that wouldn't trigger?

Reply:
Added two examples that must not trigger the message.

Reply:
Thanks!